test/k8sT/istio.go

```diff
@@ -38,7 +38,11 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sIstioTest", func() {
 		// installed.
 		istioSystemNamespace = "istio-system"
 
+<<<<<<< HEAD
 		istioVersion = "1.8.2"
+=======
+		istioVersion = "1.10.4"
+>>>>>>> istio: Update to release 1.10.4
```

Comment:
Looks like an unresolved merge conflict.

Reply:
(I usually run `git rebase --exec "make clean && make -j6 && make -C test -j6 && make -C bpf build_all -j6" origin/v1.10` locally to catch such issues.)

Reply:
Thanks for the command line Paul!

I usually runs some checks locally, but the `//go:build` introduced by Go v1.17 make it so `contrib/scripts/check-fmt.sh` fails quickly. Setting `GO_VERSION` doesn't help here since it's `gofmt`, and the script doesn't allow to override the `gofmt` binary path (e.g. to `~/sdk/go1.16.7/bin/gofmt`). Have you a workaround for this? Should we have one?